<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1191</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1191</width>
       <height>637</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone 1 - Page 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>471</width>
        <height>611</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;;
background-color: rgb(181, 206, 222);</string>
      </property>
      <property name="title">
       <string>State / City</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>271</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>300</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Zipcode</string>
       </property>
      </widget>
      <widget class="QListWidget" name="zipcodeList">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>300</y>
         <width>271</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>480</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Categories</string>
       </property>
      </widget>
      <widget class="QListWidget" name="categoryList">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>450</y>
         <width>271</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="businessGroup">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>721</width>
        <height>371</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;;
background-color: rgb(181, 206, 222);</string>
      </property>
      <property name="title">
       <string>Businesses</string>
      </property>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>651</width>
         <height>281</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>400</y>
        <width>711</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 219, 222);</string>
      </property>
      <property name="title">
       <string>Statistics</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string># of Businesses:</string>
       </property>
      </widget>
      <widget class="QListWidget" name="numBusiness">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>30</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Top Categories</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="topCategory">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>60</y>
         <width>431</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1191</width>
       <height>637</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone 2 - Page 2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1191</width>
        <height>651</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Search Business by Name</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Enter Business Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>510</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>City for Business:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bname">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>60</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;
border-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bcity">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>510</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="businesses">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>150</y>
         <width>321</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
